### PR TITLE
avoid scalable error "maxValue should be bigger than minValue" in generator dispatch 

### PR DESCRIPTION
**Pull request by dbraquart**

### CODE REVIEW

**Review by FranckLecuyer — APPROVED** [review]
Code review: OK
Tests: OK

File: src/test/java/org/gridsuite/modification/server/modifications/GenerationDispatchTest.java [hunk 1]
```diff
@@ -20,15 +21,18 @@
...
 import java.util.Map;
+import java.util.Optional;
 import java.util.UUID;
 import java.util.stream.Collectors;
 
 import static org.gridsuite.modification.server.utils.TestUtils.assertLogMessage;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertThrows;
+import static org.gridsuite.modification.server.utils.TestUtils.assertLogNthMessage;
+import static org.junit.Assert.*;
```
Comment (Walid-Sahnoun) [inline src/test/java/org/gridsuite/modification/server/modifications/GenerationDispatchTest.java:34]: no need to this chane

File: src/test/java/org/gridsuite/modification/server/utils/TestUtils.java [hunk 2]
```diff
@@ -133,29 +133,37 @@ public static String resourceToString(String resource) throws IOException {
...
+    private static Optional<String> getMessageFromReporter(String reportKey, ReporterModel reporterModel, int rank) {
         Optional<String> message = Optional.empty();
 
         Iterator<Report> reportsIterator = reporterModel.getReports().iterator();
+        int nbTimes = 0;
         while (message.isEmpty() && reportsIterator.hasNext()) {
             Report report = reportsIterator.next();
             if (report.getReportKey().equals(reportKey)) {
-                message = Optional.of(formatReportMessage(report, reporterModel));
+                nbTimes = nbTimes + 1;
```
Comment (Walid-Sahnoun) [inline src/test/java/org/gridsuite/modification/server/utils/TestUtils.java:157]: nbTimes++;

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
test ok

**Pull request merged**